files/build_templates/swss_vars.j2

```diff
@@ -6,5 +6,6 @@
     "asic_id": "{{ DEVICE_METADATA.localhost.asic_id }}",
     "mac": "{{ DEVICE_METADATA.localhost.mac }}",
     "resource_type": "{{ DEVICE_METADATA.localhost.resource_type }}",
-    "synchronous_mode": {% if DEVICE_METADATA.localhost.synchronous_mode == "disable" %}"disable"{% else %}"enable"{% endif %}
+    "synchronous_mode": {% if DEVICE_METADATA.localhost.synchronous_mode == "disable" %}"disable"{% else %}"enable"{% endif %},
+    "additional_mac": {% if DEVICE_METADATA.localhost.type == "ToRRouter" and DEVICE_METADATA.localhost.subtype == "DualToR" and (DEVICE_METADATA.localhost.hwsku == "Mellanox-SN2700-D48C8" or DEVICE_METADATA.localhost.hwsku == "Mellanox-SN2700") %}"enable"{% else %}"disable"{% endif %}
```

Comment:
this is programmed during RIF creation, why not take this mac from that flow?

Reply:
Want to clarify a flow:
Solution consists of 2 PR's: this one https://github.com/sonic-net/sonic-buildimage/pull/13546 and https://github.com/ayurkiv-nvda/sonic-sairedis/pull/1
Values defined in **swss_vars.j2** fetched in [syncd/scripts/syncd_init_common.sh:18](https://github.com/ayurkiv-nvda/sonic-sairedis/blob/master/syncd/scripts/syncd_init_common.sh#L18)

In [config_syncd_mlnx()](https://github.com/ayurkiv-nvda/sonic-sairedis/blob/master/syncd/scripts/syncd_init_common.sh#L190) SYNCD_VARS with data from **swss_vars.j2** is parsed and used to create **sai.profile** file with SAI_ADDITIONAL_MAC_ENABLED=1